app/controllers/api/v1/accounts/familiar_followers_controller.rb

```diff
@@ -12,14 +12,14 @@ def index
   private
 
   def set_accounts
-    @accounts = Account.without_suspended.where(id: account_ids).select('id, hide_collections').index_by(&:id).values_at(*account_ids).compact
+    @accounts = Account.without_suspended.where(id: account_ids).select('id, hide_collections').compact
   end
 
   def familiar_followers
     FamiliarFollowersPresenter.new(@accounts, current_user.account_id)
   end
 
   def account_ids
-    Array(params[:id]).map(&:to_i)
+    Array(params[:id]).map(&:to_i).uniq
```

Comment:
I would drop the `uniq`, ActiveRecord and/or PostgreSQL will treat it just fine.
```suggestion
    Array(params[:id]).map(&:to_i)
```